epinorm/dataFetcher.py

```diff
@@ -0,0 +1,100 @@
+import pandas as pd
+from polyglot.text import Text
+from transliterate import translit
+
+def getCountryNamesStandardised():
+    """
+    THis retrieves the country names in the file "countries", as they are in the file
+    """
+
+    df = pd.read_csv("epinorm/data/countries.csv")
+    names = df["name"]
+    return set(names)
```

Comment:
Here there is no need for an intermediate `names` variable I think.
```suggestion
    return set(df["name"])
```

Or you could even do 
```py
    return set(pd.read_csv("epinorm/data/countries.csv")["name"])
```